### ISSUE TITLE
window.devToolsExtension is obsolete for Redux Dev Tools.

### ISSUE THREAD

**Issue by ngbrown** [aspnet/JavaScriptServices#1196]
on https://github.com/aspnet/JavaScriptServices/blob/d1198aeab2a945956a611f779a42e636f645949c/templates/ReactReduxSpa/ClientApp/configureStore.ts#L12

Is obsolete and should be replaced by `window.__REDUX_DEVTOOLS_EXTENSION__` or `window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__`. 

See https://github.com/zalmoxisus/redux-devtools-extension/issues/220.

**Comment by SteveSandersonMS** [comment]
Thanks - we'll replace/remove it!

**Comment by thangchung** [comment]
@ngbrown Thank you. You are saving my day. Actually, I created the project from VS2017.4, but not actually has resolved it. I found out the fixed version at https://github.com/aspnet/templating/blob/dev/src/Microsoft.DotNet.Web.Spa.ProjectTemplates/content/ReactRedux-CSharp/ClientApp/configureStore.ts#L12, it seems like the correct one has never been pulled when we created the project from VS 2017 :)
